tests/fixtures/populate.py

```diff
@@ -114,23 +114,26 @@ def upload(uploader: Uploader):
             },
         )
 
-
-if __name__ == '__main__':
+def populate(s3_bucket: str, gs_bucket: str):
```

Comment:
this typing is incorrect.  it should be `typing.Optional[str]` since it can be `None`.